data/baseline_situations_generated/traffic_situation_02.json

```diff
@@ -1,68 +1,47 @@
 {
     "title": "CR-GW",
-    "inputFileName": "baseline_situation_02_1_ts.json",
-    "commonVector": 600.0,
     "ownShip": {
         "static": {
-            "id": "0216291f-1ce2-4e76-b752-7cf93e48f6ab",
+            "id": "6983e2ad-4f17-414d-9c07-5c12214926cc",
             "length": 122.0,
             "width": 20.0,
             "height": 8.0,
-            "speedMax": 17.0,
             "mmsi": 257847600,
             "name": "BASTO VI",
-            "shipType": "Passenger"
+            "shipType": "Passenger",
+            "speed_max": 17
         },
         "initial": {
             "position": {
-                "north": 0.0,
-                "east": 0.0,
                 "latitude": 1.02561566,
-                "longitude": 0.18309645
+                "longitude": 0.18309645,
+                "east": 0,
+                "north": 0
             },
             "sog": 5.14,
-            "cog": 0.0
+            "cog": 0.0,
+            "navStatus": "Under way using engine"
```

Comment:
new field, compared to last version.  Correct?

Reply:
should it be under_way_using_engine?

Reply:
The enum member is written with an underscore
`AISNavStatus.UNDER_WAY_USING_ENGINE`
The value of this enum member is (as currently implemented in maritime-schema) without underscore:
~~~py
class AISNavStatus(str, Enum):
    UNDER_WAY_USING_ENGINE = "Under way using engine"
~~~
Yet, this is just a very technical comment.  Whether "Under way using engine" as string is according to what COLREGs expect / require  I don't know - but you might :-)